delegation-program-leaderboard/templates/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "delegation-program-leaderboard.fullname" . }}
+  labels:
+    {{- include "delegation-program-leaderboard.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "delegation-program-leaderboard.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "delegation-program-leaderboard.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "delegation-program-leaderboard.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: DB_SNARK_HOST
+              value: {{ .Values.leaderboard.dbHost | quote }}
+            - name: DB_SNARK_PORT
+              value: {{ .Values.leaderboard.dbPort | quote }}
+            - name: DB_SNARK_USER
+              value: {{ .Values.leaderboard.dbUser | quote }}
+            - name: DB_SNARK_PWD
+              value: {{ .Values.leaderboard.dbPassword | quote }}
+            - name: DB_SNARK_DB
```

Comment:
db snark db is not very descriptive. Is that a specific db or postgres db?
I would maybe rework these into
SNARK_DB_HOST
SNARK_DB_PORT
SNARK_DB_USER
SNARK_DB_PWD
SNARK_DB_NAME
what do you think? @kaozenn

Reply:
This is application related logic. I don't have any control on it